extensions/cornerstone-dicom-rt/src/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { id } from './id';
+import React from 'react';
+
+import getSopClassHandlerModule from './getSopClassHandlerModule';
+
+const Component = React.lazy(() => {
+  return import(
+    /* webpackPrefetch: true */ './viewports/OHIFCornerstoneRTViewport'
+  );
+});
+
+const OHIFCornerstoneRTViewport = props => {
+  return (
+    <React.Suspense fallback={<div>Loading...</div>}>
+      <Component {...props} />
+    </React.Suspense>
+  );
+};
+
+/**
+ * You can remove any of the following modules if you don't need them.
+ */
+const extension = {
+  /**
+   * Only required property. Should be a unique value across all extensions.
+   * You ID can be anything you want, but it should be unique.
+   */
+  id,
+
+  /**
```

Comment:
If this isn't used, please remove it - no need to add unnecessary code, and the base cli version of the extension documents everything.